docs/UserGuide.md

```diff
@@ -206,9 +207,10 @@ Edit and updates an existing task in the task list.
 
 Examples:
 * `updatet 1 n/Meeting with TAs` Updates the name of the 1st displayed task to be `Meeting with TAs`
-* `updatet 2 n/Meeting with Prof Tan dt/2022-12-01 1200` Updates the name of the 2nd task to be `Meeting with Profs Tan` and the date to be 1st Dec 2022, 12pm.
+* `updatet 2 n/Meeting with Prof Tan dt/2022-12-01 1200` Updates the name of the 2nd displayed task to be `Meeting with Profs Tan` and the date to be 1st Dec 2022, 12pm.
+* `updatet 1 dt/2022-12-12 1200, 2022-12-12 1400` Updates the datetime of the 1st displayed task to be on `12th Dec 2022, 12-2pm.`
 
-Format: `updatet INDEX [tn/TASKNAME] [dt/DATETIME] [t/TAG]`
+Format: `updatet INDEX [tn/TASKNAME] [dt/DATETIME[, ENDDATETIME]] [t/TAG]`
```

Comment:
Maybe write change [dt/DATETIME[, ENDDATETIME]] to [dt/DATETIME, ENDDATETIME]?🤔

Reply:
Yeah, nested square brackets can be confusing, maybe I can use an asterisk and then explain that its optional